src/po/znc.ru.po

```diff
@@ -57,7 +57,7 @@ msgstr "Выйти"
 
 #: ClientCommand.cpp:51
 msgid "You must be connected with a network to use this command"
-msgstr ""
+msgstr "У вас должно быть стабильное интернет-соединения, для того чтобы использовать эту команду"
```

Comment:
1. Please fix the grammar.
2. This message has nothing to do with the stable connection. It's possible to connect with or without IRC network.